osafw-app/App_Data/template/dev/dbupdates/save/onload.js

```diff
@@ -0,0 +1,13 @@
+function highlightSQL() {
```

Comment:
duplicate code detected, just put it into separate `dev/dbupdates/highlight.js` template and include form both places